content/_redirects

```diff
@@ -3,6 +3,8 @@
 /posts/scientific-python/alt-text-workshop-summary              /scientific-python/alt-text-workshop-summary
 /posts/scientific-python/gsod-2022-proposal                     /scientific-python/gsod-2022-proposal
 /posts/numpy/mukulikapahari                                     /numpy/mukulikapahari
+/posts/networkx/outreachy-2023/intership                        /posts/networkx/outreachy-2023/intership
+/posts/networkx/outreachy-2023/contribution-phase               /posts/networkx/outreachy-2023/contribution-phase
```

Comment:
This is not necessary for new posts; I justs added this in here so the old ones can still be found at their new locations.